app/assets/javascripts/jquery_overrides.js

```diff
@@ -1,6 +1,10 @@
 jQuery.evalLog = function (text) {
   try {
-    jQuery.globalEval(text.slice('throw "error";'.length));
+    if (text.match('^throw "error";')) {
```

Comment:
@abellotti , why is this change needed? We do want that "throw error" everywhere, and probably especially when you're sending the authentication token.